Update ug (#189)
AY2021S1-CS2103T-T10-4/tp: docs/UserGuide.md

#### Adding a lesson : `add-lesson`

> If you would like to add a lesson in a class, you can make use of this command.

Reviewer: ```suggestion
If you would like to add a lesson to a class, you can make use of this command.
```

Format: `add-lesson c\CLASS_INDEX d\DAY st\START_TIME et\END_TIME v\VENUE r\NO_OF_TIMES`
* Adds a lesson at a specified `CLASS_INDEX`+ `DAY` + `START_TIME` + `END_TIME` + `VENUE` + `NO_OF_TIMES`.

For example:

> * You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02` hence 

Reviewer: ```suggestion
* You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02` from week 1 to 13. Hence,
```

> * You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02` hence 

Reviewer 2: ```suggestion
* You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02`. Hence,
```

    ![Delete lesson before](images/ugimages/DeleteLessonBefore.png)

> * You have one lesser lesson to teach now.

Reviewer: ```suggestion
* You have one less lesson to teach now.
```

> * You have one lesser lesson to teach now.

Reviewer 2: I think it should be the original.
```suggestion
* You have one less lesson to teach now.
```

#### Deleting an attendance record : `delete-attendance`

> If you would like to delete a specific attendance record from a lesson, you can make use of this command.

Reviewer: ```suggestion
If you would like to delete a specific student's attendance from a lesson, you can make use of this command.
```

Format: `delete-attendance c\CLASS_INDEX l\LESSON_INDEX s\STUDENT_INDEX w\WEEK`

* Deletes the attendance record at the specified `CLASS_INDEX` + `LESSON_INDEX` + `STUDENT_INDEX` + `WEEK`.
* The indexes **must be positive whole numbers** 1, 2, 3, …​
* The `WEEK` **must be a positive whole number** 1, 2, 3, …​

> For example"

Reviewer 2: ```suggestion
For example:
```

> * You would like to delete Alice's attendance record in Week 1. Hence, you type in the command 
> `delete-attendance c\1 l\1 s\1 w\1` and press <kbd>Enter</kbd>.

Reviewer: ```suggestion
* You would like to delete Alice's attendance for the Thursday 10.00am to 11.00am lesson of CS2103T Tutorial in Week 1.
Hence, you type in the command `delete-attendance c\1 l\1 s\1 w\1` and press <kbd>Enter</kbd>.
```

    ![Delete attendance before](images/ugimages/DeleteAttendanceBefore.png)

> * Alex's attendance record has been deleted.

Reviewer: ```suggestion
* Alice's attendance for week 1 has been deleted.
```

    ![Delete attendance after](images/ugimages/DeleteAttendanceAfter.png)

> ### Display Statistics : `stats`

Reviewer: I think every command should be part of a sub category of features?
```suggestion
#### Display Statistics : `stats`
```
